MdeModulePkg/MdeModulePkg.dsc

```diff
@@ -127,7 +127,13 @@
   HobLib|MdePkg/Library/PeiHobLib/PeiHobLib.inf
   MemoryAllocationLib|MdePkg/Library/PeiMemoryAllocationLib/PeiMemoryAllocationLib.inf
   ExtractGuidedSectionLib|MdePkg/Library/PeiExtractGuidedSectionLib/PeiExtractGuidedSectionLib.inf
+## MU_CHANGE - SmmLockBoxPeiLib only supports IA32 and X64.

```

Comment:
Later the file (in edk2 and here) has:

```
[LibraryClasses.ARM, LibraryClasses.AARCH64]
  LockBoxLib|MdeModulePkg/Library/LockBoxNullLib/LockBoxNullLib.inf
```

So, this should be unnecessary now.

Reply:
In that case the PR should be dropped right?

Reply:
Yes